tools/virtualization/mapping-a-virtualised-guest-port-to-a-host-port.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Mapping a virtualized guest port to a host port  
+subsection: virtualization  
+order: 10  
+---
+
+# Mapping a virtualized guest port to a host port
```

Comment:
This seems like it's only QEMU related. Could we maybe move it into QEMU subpage as a basic usage and then have some sub-title for the port mapping?

Reply:
Addressed here https://github.com/developer-portal/content/pull/394/commits/c8c7dbab93f7ac23cdbcb93b99cbebdb2da63799